packages/assets-controllers/src/TokensController.ts

```diff
@@ -739,12 +790,20 @@ export class TokensController extends BaseController<
     if (!suggestedAssetMeta) {
       return;
     }
-    suggestedAssetMeta.status = SuggestedAssetStatus.rejected;
-    this.hub.emit(`${suggestedAssetMeta.id}:finished`, suggestedAssetMeta);
+    const rejectedSuggestedAssetMeta = {
+      ...suggestedAssetMeta,
+      status: SuggestedAssetStatus.rejected,
+    };
```

Comment:
Same as with the previous one, I had to create a new object instead of updating status directly to prevent an error in react-native codebase.